backend/graminsta/core/serializers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: UTF-8 -*-
+
+"""
+Here defines model serializers.
+"""
+
+from django.contrib.auth import models as auth_models
+from rest_framework import serializers
+from .models import UserInfo
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """
+    UserSerializer defined for User model.
+    """
+    class Meta:
+        """
+        Meta Information
+        """
+        model = auth_models.User
+        fields = (
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+            'password',
```

Comment:
This will expose the hashed password to API callers, which is a high security risk, consider marking it as write-only.

See also: https://www.django-rest-framework.org/api-guide/serializers/#additional-keyword-arguments